### PR TITLE
Build/tox

### PR DESCRIPTION
**Pull request by issac211**
added tox config, markers and pytest config

added to the README testing Instructions

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -75,7 +75,7 @@ pydantic==1.7.3
 pyflakes==2.2.0
 pyparsing==2.4.7
 pytest==6.2.1
-pytest-asyncio<0.14.0
+pytest-asyncio==0.12.0
```
Comment (yammesicka) [inline requirements.txt:78]: Why?

File: README.md [hunk 2]
```diff
@@ -49,7 +49,14 @@ uvicorn app.main:app --reload
...
 ```shell
-python -m pytest --cov-report term-missing --cov=app tests
+# Standard tests: 'coverage' and 'flake8'
+tox
+# Only flake8 tests
+tox -e flake8
+# Coverage withe html reports
+tox -e rep
+# Marked function tests
+pytest -m MARKER_NAME
```
Comment (yammesicka) [inline README.md:59]: Leave the whole pytest line as it was before :)

File: tests/test_a_telegram_asyncio.py [hunk 3]
```diff
@@ -82,13 +82,15 @@ def gen_callback(text):
 class TestChatModel:
 
     @staticmethod
+    @pytest.mark.a_telegram_asyncio
```
Comment (yammesicka) [inline tests/test_a_telegram_asyncio.py:85]: Why have you added all these marks? We can just run `pytest tests/filename.py` if we want to ^^"

File: tox.ini [hunk 4]
```diff
@@ -1,6 +1,71 @@
...
+deps = flake8
+commands =
+    flake8 app
+    flake8 tests
+
+
+# pytest Configuration
+[pytest]
+junit_family = xunit2
+markers =
```
Comment (yammesicka) [inline tox.ini:30]: We can't allow this - I don't want a coupling that should be added every time we add a new testfile

**Review by yammesicka — CHANGES_REQUESTED** [review]
Great job! I've added few insights :)

File: README.md (hunk 2 above)
Comment (issac211) [reply README.md]: i hope adding is ok..

File: README.md [hunk 5]
```diff
@@ -52,5 +52,15 @@ uvicorn app.main:app --reload
 python -m pytest --cov-report term-missing --cov=app tests
 ```
 
+### Running tox
+```shell
+# Standard tests: 'coverage' and 'flake8'
+tox
+# Only flake8 tests
+tox -e flake8
+# Coverage withe html reports
```
Comment (yammesicka) [inline README.md:61]: fix typo: `with`

**Review by yammesicka — COMMENTED** [review]
Great job! We're ready to merge, just please fix the typo :)

File: README.md (hunk 5 above)
Comment (issac211) [reply README.md]: oops ^^"

**Pull request merged**